LB3210.org

```diff
@@ -977,6 +982,208 @@ Route route = path("users", () ->
 - Deploy in the morning, monitor your dashboards
 - However, "infra tooling" or "platform" team can be helpful
 - The same holds for the "DBA" team
+* A selection of REST patterns
+** Resource tags and caching
+*** Resources have versions
+- Servers can include an =ETag=, which specifies which /version/ of a resource is being served
```

Comment:
Consider explaining: ETag = Entity Tag, Entity~The 'thing' a Resource represents.  As fielding states it: 'More precisely, a resource R is a temporally varying membership function MR(t), which for time t maps to a set of entities, or values, which are equivalent'.